src/colvarproxy.h

```diff
@@ -101,6 +102,15 @@ class colvarproxy_atoms {
     atoms_refcount[index] += 1;
   }
 
+  /// Decrease the reference count of the given atom
+  /// \param index Internal index in the Colvars arrays
+  inline void decrease_refcount(int index)
+  {
+    if (atoms_refcount[index] > 0) {
+      atoms_refcount[index] -= 1;
+    }
```

Comment:
I was wondering why there is no way to decrease the reference count when working on https://github.com/Colvars/colvars/pull/788...